backend/services/deviceconnect/api/http/router.go

```diff
@@ -95,19 +99,19 @@ func NewRouter(
 	router.POST(APIURLInternalDevicesIDSendInventory, internal.SendInventory)
 
 	device := NewDeviceController(app, natsClient)
-	router.GET(APIURLDevicesConnect, device.Connect)
-	router.POST(APIURLInternalDevices, device.Provision)
-	router.DELETE(APIURLInternalDevicesID, device.Delete)
+	publicAPI.GET(APIURLDevicesConnect, device.Connect)
+	publicAPI.POST(APIURLInternalDevices, device.Provision)
+	publicAPI.DELETE(APIURLInternalDevicesID, device.Delete)
 
 	management := NewManagementController(app, natsClient)
-	router.GET(APIURLManagementDevice, management.GetDevice)
-	router.GET(APIURLManagementDeviceConnect, management.Connect)
-	router.GET(APIURLManagementDeviceDownload, management.DownloadFile)
-	router.HEAD(APIURLManagementDeviceDownload, management.DownloadFile)
-	router.POST(APIURLManagementDeviceCheckUpdate, management.CheckUpdate)
-	router.POST(APIURLManagementDeviceSendInventory, management.SendInventory)
-	router.PUT(APIURLManagementDeviceUpload, management.UploadFile)
-	router.GET(APIURLManagementPlayback, management.Playback)
+	publicAPI.GET(APIURLManagementDevice, management.GetDevice)
+	publicAPI.GET(APIURLManagementDeviceConnect, management.Connect)
+	publicAPI.GET(APIURLManagementDeviceDownload, management.DownloadFile)
```

Comment:
do you think we should also limit the download?

Reply:
btw there are also limits inside the mender-connect. (dont remember the defaults)